src/git_auto_export/apps.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Adding this apps.py in this format has been replaced with a more granular way defined by edX for defining the plugin-related settings.

You can read more about it in the [plugin architecture](https://github.com/openedx/edx-django-utils/tree/master/edx_django_utils/plugins).
You can see example implementations in all the applications in `src` directory. One example is [this](https://github.com/mitodl/open-edx-plugins/blob/main/src/ol_openedx_checkout_external/app.py).

To verify why you can simply install your plugin and then check `INSTALLED_APPS` in your studio/cms django shell. You won't see this package in there. Once it is configured as mentioned above you'll be able to see the plugin in INSTALLED_APPS.

Reply:
Again i am not familiar with edX plugins architecture as I have never encountered or used them before. So I followed an installable app architecture.

Anyway, will consider that way.